utils/resb/src/binary.rs

```diff
@@ -69,6 +69,7 @@ struct BinReprInfo {
 /// The `BinIndex` struct represents details of the written bundle.
 ///
 /// The index is present from [`FormatVersion::V1_1`] on.
+#[allow(dead_code)]
```

Comment:
@leftmostcat any idea why this is dead code?

Reply:
Several of these fields are only read from at present with the `serialize` feature enabled. Looks like if I run `cargo check`, I get a warning about that which I don't get with `cargo check --all-features`.

Reply:
Ok, can you fix it in your follow-up PR by gating this on `#[cfg(feature = "serialize")]` or whichever feature needs it?